src/pages/suinJung/Main/Component/Article.js

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import CommentList from './CommenList';
+
+function Article({ name }) {
+  const [content, setContent] = useState('');
+  const [contentArray, setContentArray] = useState([]);
+
+  const commentContent = event => {
+    setContent(event.target.value);
+  };
+
+  const handleClick = e => {
+    e.preventDefault();
+    contentArray.push(content);
+    setContentArray(contentArray);
+    setContent('');
```

Comment:
```suggestion
    setContentArray([...contentArray, content]);
    setContent('');
```
state에 push해서 직접 값을 변화시키지 말고 setState 함수를 쓰는 게 더 안정적일 것 같아요.
setContent("");로 state 값을 다시 변경하기보다, input의 값 자체를 ""로 변경시켜주면 commentContent 함수가 작동해서 state도 값이 ""가 되니까 그렇게 해주시면 어떨까요?